httpclient/httpclient.go

```diff
@@ -84,6 +100,15 @@ func New(cfg Config) (runtime *httptransport.Runtime, err error) {
 		transport = &roundTripperWithSourceChannel{OriginalRoundTripper: transport, SourceChannel: sourceChannel}
 	}
 
+	if cfg.Profile().OrganizationID != "" || cfg.Profile().ProjectID != "" {
```

Comment:
Let's make sure we have validation that these are always set together. Both populated, or both empty.